contrib/cirrus/setup_container_environment.sh

```diff
@@ -5,15 +5,12 @@ source $(dirname $0)/lib.sh
 
 req_env_var GOSRC OS_RELEASE_ID CONTAINER_RUNTIME
 
-DIST=$OS_RELEASE_ID
-IMAGE=${DIST}podmanbuild
-
 # Since CRIU 3.11 has been pushed to Fedora 28 the checkpoint/restore
 # test cases are actually run. As CRIU uses iptables to lock and unlock
 # the network during checkpoint and restore it needs the following two
 # modules loaded.
 modprobe ip6table_nat || :
 modprobe iptable_nat || :
 
-# Build the test image
-${CONTAINER_RUNTIME} build -t ${IMAGE} -f Dockerfile.${DIST} .
+# Pull the test image
+${CONTAINER_RUNTIME} pull quay.io/libpod/in_podman:latest
```

Comment:
Is it possible to define `quay.io/libpod/in_podman:latest` in `lib.sh` then reference the variable instead?  That would be a lot cleaner and easier to maintain.

Reply:
if you really want to twist my arm on that, yes

Reply:
/me gets out his plush wrench...puts it back in it's hiding spot, and fetches the iron one :grin: